ship_it/__init__.py

```diff
@@ -30,6 +36,10 @@ def fpm(manifest_path, requirements_file_path=None, setup_py_path=None,
 
     man_args, man_flags = manifest.get_args_and_flags()
     man_flags.extend(overrides.items())
+
+    if not any(flag[0] == 'version' for flag in man_flags):
```

Comment:
this probably _is_ the best way to make this check, but I may refactor a bit after merging to make this cleaner